lib/create-icon-set-from-fontawesome5.js

```diff
@@ -49,14 +54,77 @@ export function createFA5iconSet(glyphMap, proVersion = false) {
     'Font Awesome 5 Brands'
   );
 
-  function getIconSetForProps(props) {
+  function iconSetFromStyle(style) {
+    switch (style) {
+      case FA5Style.brand:
+        return BrandsSet;
+      case FA5Style.light:
+        return LightSet;
+      case FA5Style.solid:
+        return SolidSet;
+      default:
+        return RegularSet;
+    }
+  }
+
+  function iconSetFromFamily(family) {
+    switch (family) {
+      case 'brands':
+        return BrandsSet;
+      case 'regular':
+        return LightSet;
+      case 'solid':
+        return SolidSet;
+      default:
+        return RegularSet;
+    }
+  }
+
+  function styleFromProps(props) {
     const { brand, light, solid } = props;
 
-    if (brand) return BrandsSet;
-    if (light) return LightSet;
-    if (solid) return SolidSet;
+    if (brand) return FA5Style.brand;
```

Comment:
Thought we were able to remove the `brand` prop with this change?

Reply:
This is related to my last comment 👇🏻